js/DropView

```diff
@@ -0,0 +1,79 @@
+import Adapt from 'core/js/adapt';
+import a11y from 'core/js/a11y';
+
+export default class DropView extends Backbone.View {
+
+  className() {
+    let classes = 'drop is-closed';
+    const modelClasses = this.model.get('_drop')._classes;
+
+    if (modelClasses) classes += ` ${modelClasses}`;
+
+    return classes;
+  }
+
+  initialize({ $content }) {
+    this.$content = $content;
+    this.listenTo(Adapt, 'remove', this.remove).render();
```

Comment:
```suggestion
    this.listenTo(Adapt, 'remove', this.remove);
    this.render();
```

Reply:
Second option looks a bit cleaner to me.